src/docs-assembler/navigation.yml

```diff
@@ -322,16 +320,20 @@ toc:
                     # Children include the entire AsciiDoc book
 
           # Beats
-          # 🔜 https://github.com/colleenmcginnis/beats/blob/migrate-docs/docs/reference/toc.yml
+          # ✅ https://github.com/elastic/beats/blob/main/docs/reference/toc.yml
           - toc: beats://reference
             path_prefix: reference/beats
             # Children include all entire AsciiDoc books: Beats, Auditbeat,
             # Filebeat, Heartbeat, Metricbeat, Packetbeat, Winlogbeat,
             # Elastic logging plugin for Docker
 
+          # 📝 TO DO: Delete this item
+          # This should be replaced by `elasticsearch://reference/enrich-processor` below
+          - toc: elasticsearch://reference/ingestion-tools/enrich-processor
+            path_prefix: reference/elasticsearch/enrich-processor
+
           # Processor reference
-          # 📝 TO DO: Directory depth in elasticsearch repo: would require toc 2 levels down
-          # 🔜 https://github.com/colleenmcginnis/elasticsearch/blob/docs-assembler-prep/docs/reference/enrich-processor/toc.yml
+          # ✅ https://github.com/elastic/elasticsearch/blob/main/docs/reference/enrich-processor/toc.yml
           - toc: elasticsearch://reference/enrich-processor
             path_prefix: reference/enrich-processor
```

Comment:
I would expect we'd be able to remove `elasticsearch://reference/ingestion-tools/enrich-processor` in favor of `elasticsearch://reference/enrich-processor` now that https://github.com/elastic/elasticsearch/pull/125118 was merged, but at least when I run `build-all` locally I get errors if I delete this.